azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -751,6 +770,26 @@ private ImmutableMap getLabelsForPod(final int executionId) {
     return mapBuilder.build();
   }
 
+  /**
+   * Get a {@code lableSelector} that can be used to list all the flow-container-pods for the
+   * current namespace.
+   *
+   * @return label selector
+   */
+  public static String getLabelSelector(Props azkProps) {
```

Comment:
Can you please use final wherever possible? Applies to rest of the changes in this PR.

Reply:
Done.